tests/handlers/test_web.py

```diff
@@ -114,6 +115,10 @@ def get_variables(cls):
         def get_static_root(cls):
             return static_root
 
+        @classmethod
+        def get_routers(cls):
```

Comment:
It's a bit strange that you overrides this method and than patch it below.
Why not to declare at the beginning of the test function:

auth_router = mocker.MagicMock()
no_auth_router = mocker.MagicMock()

then make this method return this two mock objects ?